DEMOQA/cypress/tests/login.js

```diff
@@ -0,0 +1,204 @@
+describe('login', () => {
+
+    const testData = require('../fixtures/testData.json')[0]
+    beforeEach(() => {
+        cy.visit("https://demoqa.com/automation-practice-form")
+    });
+    it('submit successfully with valid input', () => {
+        cy.fixture('testData').then((data) => {
+            cy.get("#firstName").type(data.firstName)
+            cy.get("#lastName").type(data.lastName)
+            cy.get('#userEmail').type(data.userEmail)
+            cy.get("label[for='gender-radio-1']").click();
```

Comment:
You can use the attribute `value`, store the gender value in fixture file and then you can do this:
```js
cy.get(`value="${data.gender}"`).click()
```